src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.component.ts

```diff
@@ -722,15 +722,26 @@ export class OnlineGameWrapperComponent extends GameWrapper implements OnInit, O
             }
         }
     }
-    public getPlayerNameFontColor(player: number): { [key: string]: string} {
-        if (this.observerRole === player || this.observerRole > 1) {
-            return { color: 'black' };
-        } else if (this.opponentIsOffline()) {
-            return this.OFFLINE_FONT_COLOR;
+    public getPlayerNameClass(player: number): string {
+        if (this.opponentIsOffline()) {
+            return 'is-grey-light';
         } else {
-            return { color: 'black' };
+            if (player === 0) {
+                return 'is-white';
+            } else {
+                return 'is-black';
+            }
         }
     }
+    public getBoardHighlight(): string[] {
+        if (this.observerRole != null && this.isCurrentPlayer()) {
```

Comment:
"this.observerRole != null" est vraiment utile ?
J'me souviens plus si il peut être nul avant ngOnInit ou constructor.

Reply:
oui, il est nul à l'initialisation de la page !